antora.yml

```diff
@@ -35,6 +35,7 @@ asciidoc:
       https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md
```

Comment:
We should also update the `kogito_sw_examples_url` link

Reply:
Updated to https://github.com/kiegroup/kogito-examples/tree/1.32.x/serverless-workflow-examples